4-css-grid/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+    display: grid;
+    gap: 5px;
+    grid-template-areas: 'boxa boxa boxb'
+                        'boxc boxd boxb';
+}
+#boxa, #boxb, #boxc, #boxd {
+    background-color: rgb(100, 100, 100);
+    border-radius: 5px;
+    color: white;
+    padding: 20px;
+}
+#boxa {
+    grid-area: boxa;
+    /* 
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 1;
+    grid-row-end: 2; */
```

Comment:
Remove commented code, I see you did it with columns and rows, as we expected, but don't keep unused code in your files.